app/scripts/modules/core/src/cluster/allClustersGroupings.component.ts

```diff
@@ -1,14 +1,7 @@
 import { module } from 'angular';
+import { react2angular } from 'react2angular';
+import { AllClustersGroupings } from './AllClustersGroupings';
```

Comment:
The more I read the name of this component, the more I hate it. No good alternative comes to mind though.